packages/mongo-nav/src/mongo-select/MongoSelect.tsx

```diff
@@ -37,11 +37,14 @@ const menuItemContainerStyle = css`
 
 const ulStyle = css`
   max-height: ${5 * menuItemHeight}px;
-  overflow: scroll;
```

Comment:
Does just doing `overflow-y: auto` here solve this without adding the conditional css classes?

Reply:
Didn't think of that, let me try

Reply:
Lol yes, much simpler. Thank you